### PR TITLE
use `component-base` to operator flag

### PR DESCRIPTION
**Pull request by huiwq1990**
This PR use `component-base` to operator flag

### CODE REVIEW

**Comment by Jefftree** [comment]
Hello thank you for your PR. It is already possible to override the default logging level by passing in a `--v` parameter.  I'm curious about why this change is needed?

**Comment by huiwq1990** [comment]
@Jefftree sorry, my mistake. The flag `--v` work fine. I change my PR, use  `component-base` to operator flag.

File: cmd/server/main.go [hunk 1]
```diff
@@ -66,10 +67,12 @@ func main() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "error setting klog flags: %v", err)
 	}
-	local.VisitAll(func(fl *flag.Flag) {
-		fl.Name = util.Normalize(fl.Name)
-		flags.AddGoFlag(fl)
-	})
+	flags.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	flags.AddGoFlagSet(local)
```
Comment (jdnurme) [inline cmd/server/main.go:71]: Can you run `./bin/proxy-server --help` before and after your change, and provide the diff to this PR? We want to makes sure we aren't clobbering flags.

File: cmd/agent/main.go [hunk 2]
```diff
@@ -49,10 +51,12 @@ func main() {
...
 		fmt.Fprintf(os.Stderr, "error setting klog flags: %v", err)
 	}
-	local.VisitAll(func(fl *flag.Flag) {
-		fl.Name = util.Normalize(fl.Name)
-		flags.AddGoFlag(fl)
-	})
+	flags.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	flags.AddGoFlagSet(local)
+
+	logs.InitLogs()
```
Comment (jdnurme) [inline cmd/agent/main.go:57]: Can you run `./bin/proxy-agent --help` before and after your change, and provide the diff to this PR? We want to makes sure we aren't clobbering flags.

**Review by jdnurme — CHANGES_REQUESTED** [review]

**Pull request closed without merging**